lib/actions/local/ssh.coffee

```diff
@@ -1,113 +0,0 @@
-###
```

Comment:
This file implemented `balena local ssh` (as opposed to `balena ssh`), but since CLI v11.0.0 it it no longer available:
```
$ ./bin/balena local ssh
Note: the command "balena local ssh" was replaced in CLI version v11.0.0.
Please use "balena ssh" instead.
```
I don't remember why it wasn't deleted before: probably just forgotten.